extensions/amp-analytics/amp-analytics.md

```diff
@@ -685,6 +685,49 @@ Referrer policy is only available for `image` transport. If `referrerPolicy: no-
 }
 ```
 
+#### Linkers
+The `linkers` configuration object is used to enable cross domain ID syncing. `amp-analytics` will use this configuration to create an unique query parameter that can be used to join sessions. This parameter will then be appended to the specified outgoing links.
+
+```javascript
+"linkers": {
+  <paramName>: {
+    ids: <Object>,
+    proxyOnly: <boolean>,
+    destinationDomains: <Array<string>>,
+    enabled: <boolean>
+  }
+}
+```
+
+- `paramName` - This user defined name determines the name of the created query parameter.
+- `ids` - An object containing key-value pairs that will be passed along.
+- `proxyOnly` - (optional) Flag indicating whether the links should only be decorated on pages served on a proxy origin. Defaults to `true`.
+- `destinationDomains` - (optional) Links will be decorated if their domains are included in this array. Defaults to `canonical` and `source` domains.
+- `enabled` - Publishers must explicity set this to `true` to opt-in to using this feature.
+
+Example:
+```javascript
+"linkers": {
+  "linker1" : {
+    "ids" : {
+      "cid": "CLIENT_ID(_ga)",
+      "uid" "QUERY_PARAM(uid)",
+    }
+    "proxyOnly" : false,
+    "enabled": true,
+  }
+}
+```
+
+The above config would generate a parameter in this structure: `linker1=1~1clz2wh~cid~amp-SlB2gfii_11dWqtSr3zT4w~uid~1234`.
```

Comment:
Please also give a structure of the string.
```
<paramName>=<version>~<checkSum>~<idName1>~<idValue1>~<idName2>~<idValue2>....
```

Reply:
Done.